server/src/main/java/org/elasticsearch/cluster/coordination/UnsafeBootstrapMasterCommand.java

```diff
@@ -57,12 +54,10 @@
             "\n" +
             "Do you want to proceed?\n";
 
-    static final String NOT_MASTER_NODE_MSG = "unsafe-bootstrap tool can only be run on master eligible node";
+    public static final String NOT_MASTER_NODE_MSG = "unsafe-bootstrap tool can only be run on master eligible node";
```

Comment:
Why do we need to change the accessibility? In the ES version they're still package private.

Reply:
Good point. Our `UnsafeBootstrapAndDetachCommandIT` was located in `io.crate.node` already before, therefore it was not in the same package. I guess lets just move the test over to `org.elasticsearch.cluster.coordination`.